lib/content-services/src/lib/search/components/search-check-list/search-check-list.component.scss

```diff
@@ -1,4 +1,5 @@
-.adf-search-check-list > div {
+.adf-search-check-list > fieldset {
+    border: 0 none;
```

Comment:
I think you don't need to apply both 0 and none. If border has 0 width then it is invisible. If border is none then it is invisible too. So no need to apply both probably.